docs/doxygen/ie_docs.xml

```diff
@@ -211,6 +211,8 @@ limitations under the License.
                 <tab type="user" title="NonZero-3" url="@ref openvino_docs_ops_condition_NonZero_3"/>
                 <tab type="user" title="NormalizeL2-1" url="@ref openvino_docs_ops_normalization_NormalizeL2_1"/>
                 <tab type="user" title="NotEqual-1" url="@ref openvino_docs_ops_comparison_NotEqual_1"/>
+                <tab type="user" title="NV12toBGR-8" url="@ref openvino_docs_ops_image_NV12toBGR_8"/>
+                <tab type="user" title="NV12toRGB-8" url="@ref openvino_docs_ops_image_NV12toRGB_8"/>
```

Comment:
Just curios. Will we have `BGRtoRGB` operation or can it be already represented in nGraph? If yes, the `NV12toRGB-8` can be represented as a sequence of `NV12toBGR-8 -> BGRtoRGB` and allows to implementation to fuse them if supported. IMHO, this is more nGraph way.

Reply:
Good question, we're going to have RGB<->BGR conversion in future. @jane-intel, can you please share your thoughts?

Reply:
1. We won't have RGB->BGR, we already have it through Gather operation or Split->Concat if you wish. No need to extend opset with those.
2. Leaving one operation `NV12toBGR` will make us provide a transformation to fuse `NV12toBGR->Gather` to `SuperPrivateNV12toRGB` for all the plugins, so we 1 -- won't reduce binary size and 2 -- will extend load time with yet another transformation.

Reply:
In this case should we add `NV12toRGBp` to have fused verison of `NV12toRGB` and `transpose`?